content/docs/command-reference/data/status.md

```diff
@@ -0,0 +1,133 @@
+# data status
+
+Show changes in the data tracked by DVC in the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc data status [-h] [-q | -v]
+                       [--granular] [--unchanged]
+                       [--untracked-files [{no,all}]]
+                       [--json]
+```
+
+## Description
+
+The `data status` command displays the state of the working directory and the
+committed changes with respect to the last Git commit (`HEAD`).
```

Comment:
```suggestion
changes with respect to the last Git commit (`HEAD`).
```

Very minor suggestion (not a blocker) to get rid of "committed" unless it's absolutely needed. I think it's easy to overuse these words until they lose all meaning.

Reply:
I was trying to say that the first part, pertains to the "DVC uncommitted changes" and the next is related to the `"DVC committed changes".  But I think we could drop "committed" entirely.